src/main/java/uk/gov/hmcts/reform/civil/controllers/cases/CasesController.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.hmcts.reform.civil.controllers.cases;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.civil.helpers.CaseDetailsConverter;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.service.CoreCaseDataService;
+
+import java.math.BigDecimal;
+
+@Api
+@Slf4j
+@RestController
+@AllArgsConstructor
+@RequestMapping(
+    path = "/cases",
+    produces = MediaType.APPLICATION_JSON_VALUE
+)
+public class CasesController {
+
+    private final CoreCaseDataService coreCaseDataService;
+    private final CaseDetailsConverter caseDetailsConverter;
+
+    @GetMapping(path = {
+        "/{cid}",
+    })
+    @ApiOperation("get case by id from CCD")
+
+    public ResponseEntity<CaseData> getClaimById(
+        @PathVariable("cid") Long claimId
```

Comment:
Do we want to add AUTHORIZATION Header in the request?

Reply:
Good question, In the getcases we're using with user.getUserName and user.getPassword from application.yml
`    public CaseDetails getCase(Long caseId) {
        String userToken = idamClient.getAccessToken(userConfig.getUserName(), userConfig.getPassword());
        return coreCaseDataApi.getCase(userToken, authTokenGenerator.generate(), caseId.toString());
    }`
**I did add nothing in here, do I need to change it ?**

- This explain why I don't add the AUTHORIZATION explicit in the code, but when call from post man we need to add AUTHORIZATION this is a requirement of springboot-security.

Reply:
I made a change code in this point I created a new method with auth field